sklearn/feature_selection/_rfe.py

```diff
@@ -625,7 +645,27 @@ def fit(self, X, y, groups=None):
         self.estimator_ = clone(self.estimator)
         self.estimator_.fit(self.transform(X), y)
 
-        # Fixing a normalization error, n is equal to get_n_splits(X, y) - 1
-        # here, the scores are normalized by get_n_splits(X, y)
-        self.grid_scores_ = scores[::-1] / cv.get_n_splits(X, y, groups)
+        # reverse to stay consistent with before
+        scores_rev = scores[:, ::-1]
+        self.cv_results_ = {}
+        self.cv_results_["mean_score"] = np.mean(scores_rev, axis=0)
+        self.cv_results_["std_score"] = np.std(scores_rev, axis=0)
```

Comment:
I am wondering if we should not use `mean_test_score` / `std_test_score` / `split{i}_test_score` instead.

1. this would be more consistent with `GridSearchCV`
2. this would be more future proof in case we want to add an option to also compute the training score in RFECV although this is probably a case of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).

**Edit** actuall ignore that remark. #16392 already uses the `mean_score` notation...

Reply:
Being consistent with `GridSearchCV` looks like a net win to me. I am okay with deprecating `mean_score` in `GraphicalLassoCV` and switch to using `*_test_score` instead.

@ogrisel Do you think deprecating would be too much of a hassle?

Reply:
I am fine with both approaches.